### ISSUE TITLE
Context in subscriptions

### ISSUE THREAD

**Issue by XenorPLxx** [rmosolgo/graphql-ruby#1052]
Hey!
At first I wanted to thank you for your great work!

I've got a bit of a problem understanding how passing context works when using GraphQL subscriptions.

I've got `current_user` stored in my context and my context is fetched in `GraphQL::Subscriptions#execute` (after `GraphQL::Subscriptions#trigger` being called by me) by using `GraphQL::Subscriptions#read_subscription` method. I didn't override `GraphQL::Subscriptions#execute` in any way.

`GraphQL::Subscriptions#execute` then calls `GraphQL::Schema#execute` and it's where context is stripped and only `:backtrace` and `:tracers` are used to call `GraphQL::Schema#multiplex`.

How can I pass `current_user` all the way to my `:field` definition in SubscriptionType so I can use it in `resolve`?

I would like to be able to use it like that:
```
    resolve ->(_obj, _args, ctx) {
      SomeModel.where(user: _ctx[:current_user])
    }
```

**Comment by rmosolgo** [comment]
Hi! The code sample you posted looks right to me, I expect it to work as you described. What happened when you tried it?

**Comment by XenorPLxx** [comment]
Wow. I believe it's one of those 'talking to yellow duck' situations (like when something doesn't work, you tell someone about it and then problem automagically resolves in your head). 


It works as expected now!


My mistake was that I've tried to invoke `_ctx.current_user` instead of `_ctx[:current_user]`. And then when I wrote this post I've wrote the correct code sample instead of wrong one :D.


Sorry for bothering you and thank you for your quick response!
